### ISSUE TITLE
Missed provide ENTITY_METADATA_TOKEN in EntityDataModuleWithoutEffects

### ISSUE THREAD

**Issue by alexrutkov** [ngrx/platform#3541]
### Minimal reproduction of the bug/regression with instructions

Should add in [entity-data-without-effects.module.ts](https://github.com/ngrx/platform/blob/master/modules/data/src/entity-data-without-effects.module.ts)
`{
          provide: ENTITY_METADATA_TOKEN,
          multi: true,
          useValue: config.entityMetadata ? config.entityMetadata : [],
        },
 `
as in [entity-data.module.ts](https://github.com/ngrx/platform/blob/master/modules/data/src/entity-data.module.ts)

### Minimal reproduction of the bug/regression with instructions

Without this provide I get error 
`No EntityDefinition for entity type '*'`

### Versions of NgRx, Angular, Node, affected browser(s) and operating system(s)

NgRx/data@lastest

### I would be willing to submit a PR to fix this issue

- [X] Yes
- [ ] No

**Comment by timdeschryver** [comment]
Hi @alexrutkov it's hard to follow here.
What are you trying to do that resolves in this error?

If I'm not mistaken, the `ENTITY_METADATA_TOKEN` token is missing from `EntityDataModuleWithoutEffects` ?

**Comment by alexrutkov** [comment]
I want use just EntityDataModuleWithoutEffects, but can't...get a lot mistakes.
Start use native Store.

**Comment by timdeschryver** [comment]
Closing this one because it has no reproduction.
Feel free to reopen or recreate with a reproduction.
